pkg/controllers/hub/trafficmanagerbackend/suite_test.go

```diff
@@ -54,7 +55,9 @@ func TestAPIs(t *testing.T) {
 }
 
 var _ = BeforeSuite(func() {
-	klog.SetLogger(zap.New(zap.WriteTo(GinkgoWriter), zap.UseDevMode(true)))
+	logger := zap.New(zap.WriteTo(GinkgoWriter), zap.UseDevMode(true))
+	klog.SetLogger(logger)
+	log.SetLogger(logger)
```

Comment:
Why do we need this change?

Reply:
the log complains that the setLogger is never called. so i made this change